Jenkinsfile

```diff
@@ -265,7 +265,7 @@ pipeline {
                     env.COMMIT_MESSAGE.split('\n').each { line ->
                         String key, value
                         try {
-                            (key, value) = line.split(':')
+                            (key, value) = line.split(':', 2)
```

Comment:
This is obsolete now.  You need to replace the entire block: https://github.com/daos-stack/daos/blob/1a149dd13cf39c0b861686c6529d6bcc4c9c6902/Jenkinsfile#L260-L279 with simply:

```
                pragmasToEnv()
```